addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -300,6 +300,7 @@ Many2ManyTagsField.supportedTypes = ["many2many"];
 Many2ManyTagsField.fieldsToFetch = {
     display_name: { name: "display_name", type: "char" },
 };
+Many2ManyTagsField.additionalClassNames = ["d-inline-flex", "mw-100"];
```

Comment:
this might not be necessary anymore with https://github.com/odoo/odoo/pull/96865

Reply:
Remove this mechanism if https://github.com/odoo/odoo/pull/96865 fixes the issue

Reply:
it didn't fix the issue because the css class is not the same as in legacy.
It was `o_field_many2manytags` in legacy and it's now `o_field_many2many_tags` with an _ between m2m and tags.